### DISCUSSION TITLE
Relicense to AGPL?

### DISCUSSION THREAD

**Discussion by upsuper** [navidrome/navidrome#2235] [General]
This is an application that provides service over network, and using GPL is probably not going to help pushing commercial users to contribute back. I guess it makes more sense for it to be licensed in [AGPL](https://en.wikipedia.org/wiki/GNU_Affero_General_Public_License)?

**Comment by deluan** [comment]
I don't think a license would force anyone to contribute back (even though it would be required). I used AGPL in the past (https://github.com/navidrome/navidrome/commit/ce863f0f353fdf818e1eb293b2e06eae34ec68ef) but ended up changing it to GPL, don't remember exactly why.

I'm interested in hearing from others what are their opinions on this (specially who downvoted it, @Igorgro?)

**Reply by upsuper** [reply]
It probably would force them contribute back, but at least it can push them to publish their code rather than comfortably profiting from open source work with no cost.

**Comment by Igorgro** [comment]
> I don't think a license would force anyone to contribute back

Exactly) Moreover, the change of license often cause a lot of controversies in the community, so it is better to keep this as it is unless there are any major need.
